File: articles/trust-cooperation-economic-systems/go/trust_cooperation_welfare.go
package main
import "fmt"
func main() {
	welfare := 0.70 + (0.30*0.80 + 0.25*0.75 + 0.20) - 0.05 - (0.05 * 0.80)
	fmt.Printf("Synthetic trust and cooperation welfare: %.3f\n", welfare)
}

File: articles/trust-cooperation-economic-systems/fortran/trust_cooperation_welfare.f90
program trust_cooperation_welfare
  implicit none
  real :: welfare
  welfare = 0.70 + (0.30 * 0.80 + 0.25 * 0.75 + 0.20) - 0.05 - (0.05 * 0.80)
  print *, "Synthetic trust and cooperation welfare:", welfare
end program trust_cooperation_welfare
